cypress/e2e/unit_tests/uninstall.spec.ts

```diff
@@ -17,8 +17,15 @@ describe('Epinio uninstallation testing', () => {
     cy.epinioUninstall();
   });
 
-  it('Remove the Epinio helm repo', () => {
-    cy.removeHelmRepo();
-  });
-
+  if (Cypress.env('experimental_chart_branch') != null) {
```

Comment:
We use just one `it('Add the Epinio helm repo')` block so why we have two for `remove`? Please unify.

Reply:
Because it allows to show the test name differently.

I actually wanted to have 2 separate ones for the addition, one that pointed to the regular test (`Add the Epinio helm repo`) and another one for experimental (`Add the Epinio experimental helm repo`). I actually dropped this idea and unified them into 1 single sole test because other subsequent tests were affected and was messy.

But ok I see your point and we should be consistent, I will change it.